docs/development.md

```diff
@@ -245,3 +245,11 @@ vagrant up centos7-dynflow-devel
 ```
 
 In the vagrant box, the dynflow repository is cloned to `/home/vagrant/dynflow`.
+
+## Non vagrant use
+
+Make sure your /etc/sudoers contains following, otherwise the installer will fail, on some platforms it's commented by default
+
+```
+includedir /etc/sudoers.d
+```
```

Comment:
Hm, on my Fedora 27 this lead to syntax error, my sudoers explicitly says that # is not a comment:
```
## Read drop-in files from /etc/sudoers.d (the # here does not mean a comment)
#includedir /etc/sudoers.d
```

Reply:
ok, removed as I see new OSes contains that already (with '#')